02/app.js

```diff
@@ -13,16 +13,28 @@ function init() {
 }
 
 function setRandomPosition(element, error = null) {
-    element.style.top = Math.random() * 600 + 'px';
-    element.style.left = Math.random() * 800 + 'px';
-
-    if(error) {
-        throw error;
-    }
-}
+    try {
```

Comment:
To przechwycenie powinno być wykonane w momencie wywołania `setRandomPosition(...)` czyli w okolicy 26/27 linii